invenio_app_rdm/theme/views.py

```diff
@@ -35,11 +35,13 @@ def create_blueprint(app):
     @blueprint.before_app_first_request
     def init_menu():
         """Initialize menu before first request."""
-        item = current_menu.submenu("main.deposit")
-        item.register(
-            "invenio_app_rdm_records.dashboard", _("My Dashboard"), order=1)
+        current_menu.submenu('actions.deposit').register(
+            'invenio_app_rdm_records.deposit_search',
```

Comment:
Is the name of the submenu (`invenio_app_rdm_records.deposit_search`) here a leftover or did you agree to rename it with @ntarocco ?

Reply:
Discussed, it is ok like this. To be changed in the next PR.